ocaml/idl/datamodel.ml

```diff
@@ -6281,6 +6281,11 @@ module Message = struct
         [(Ref _message, "self", "The reference of the message to destroy")]
       ~flags:[`Session] ~allowed_roles:_R_POOL_OP ()
 
+  let destroy_many =
+    call ~name:"destroy_many" ~in_product_since:rel_next
```

Comment:
Please omit `~in_product_since:rel_next`.

Reply:
This causes:

Reply:
Instead, add `~lifecycle:[]`.